.vsts-ci/vsts-arm-ci.yml

```diff
@@ -27,10 +27,21 @@ jobs:
     preview: true
     useacr: true
 
-- template: phase.yml
+- template: .vsts-ci/phase.yml
   parameters:
-    name: centos7
```

Comment:
So where is centos?

Reply:
Centos is in the normal CI.  This was just added to verify that the acr flag was working as expected.